.github/workflows/ci.yml

```diff
@@ -115,5 +114,36 @@ jobs:
           wget -qO- get.nextflow.io | bash
           sudo mv nextflow /usr/local/bin/
 
-      - name: Run pipeline with test data
```

Comment:
Just making sure: The `test.yml` files make these lines (118,119) obsolete? Or were they removed by accident?

Reply:
Yes, the matrix run is not necessary anymore as now, the different test profiles are run by pytest